CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 2.34.0-rc.0 / 2022-02-24
+
+* [CHANGE] UI: Classic UI removed. #10208
+* [FEATURE] Tracing: OpenTelemetry based tracing. #9724, #10203, #10276
```

Comment:
Does #10338 need to be included in this list?

Reply:
My thought with not including 10338 is that it only changed internal code/some cleanup whereas the other three changed user behavior/configuration. I would be ok adding it if you think it still makes sense though.

Reply:
Ok sounds good, we can keep it this way.